src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../essentials';
+import { Box } from '../Box/Box';
+import Compact from './components/Compact';
+import DefaultPanel from './components/Default';
+
+interface AccordionProps {
+    heading?: string;
+    description?: string;
+    info?: string;
+    buttonLabel?: string;
+    variant?: 'compact' | 'default';
+    expanded?: boolean;
+    children: ReactNode;
+}
+
+const HorizontalDivider = styled(Box)`
+    border: 0;
+    border-top: solid 1px ${Colors.AUTHENTIC_BLUE_200};
+`;
+
+const HorizontalDividerTop = HorizontalDivider;
+
+const HorizontalDividerBottom = styled(HorizontalDivider)`
+    display: none;
+`;
+
+const RenderedSection = styled(Box)`
+    :last-child ${HorizontalDividerBottom} {
+        display: inherit;
+    }
+`;
+
+const Accordion = ({ heading, description, info, buttonLabel, variant, expanded, children }: AccordionProps) => (
+    <RenderedSection role="group">
+        <HorizontalDividerTop />
+        {variant === 'compact' ? (
```

Comment:
I'd use an enum for the variants so that we don't need to write the constants manually each time, the goal would be to  just do `variant === AccordionVariants.Compact`

Reply:
Got your point but this is the only part `variant` is validated, so I prefer to leve as a string